client/src/components/Anonymous.jsx

```diff
@@ -170,6 +173,61 @@ const Anonymous = ({
 		});
 	};
 
+	const reportUser = async () => {
+		// Get the other user id 
+		const otherUserId = state[currentChatId]?.userIds.find(
+			id => id !== authState.loginId && id !== authState.email
+		  );
+
+		  if (!otherUserId) {
+			return { success: false, message: "Could not identify the user to report" };
```

Comment:
This isn't a report feature, so the message should be, `could not find user to block`

just because a user blocks someone doesn't mean they want to report them